### PR TITLE
bug: 353505 - Fix pseudonym sort order

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: test/unit/repos/pseudonyms.test.js [hunk 1]
```diff
@@ -44,6 +45,55 @@ describe('Pseudonyms repo', () => {
...
+        username: 'Kendrick_Wuckert'
+      },
+      {
+        rowKey: 4,
+        timestamp: '2024-04-11T08:01:24.8904697Z',
+        username: 'Shane63'
+      }
+    ]
+
+    describe('sortByTimestamp', () => {
```
Comment (jbarnsley10) [inline test/unit/repos/pseudonyms.test.js:72]: The only thing I'd suggest is having your timestamps in non-linear order in the test data. Currently, if no sorting occurs, your 'ascending' test passes

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
I really like your code here. Just the one change

File: test/unit/repos/pseudonyms.test.js (hunk 1 above)
Comment (whitewaterdesign) [reply test/unit/repos/pseudonyms.test.js:72]: good spot, thanks

**Review by jbarnsley10 — APPROVED** [review]
Perfect

**Pull request merged**